service/src/main/java/com/zebrunner/reporting/service/TestRunArtifactService.java

```diff
@@ -0,0 +1,51 @@
+package com.zebrunner.reporting.service;
+
+import com.zebrunner.reporting.domain.db.TestRunArtifact;
+import com.zebrunner.reporting.persistence.dao.mysql.application.TestRunArtifactMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class TestRunArtifactService {
```

Comment:
Do we need all of those methods? We should probably double-check with @vdelendik, we initially agreed to support an ability to attach artifacts only